packages/types/src/clerk.ts

```diff
@@ -996,6 +1001,11 @@ export type OrganizationSwitcherProps = CreateOrganizationMode &
      * the number of max allowed members is equal to 1
      */
     skipInvitationScreen?: boolean;
+    /**
+     * Hides the slug field in the form.
+     * @default undefined
+     */
+    hideSlug?: boolean;
```

Comment:
@wobsoriano We need to add this to OrganizationList as well, you can access the "CreateOrganizationForm" from that component as welll

Reply:
Thanks @panteliselef, I haven't really tried playing with `OrganizationList` component and forgot that we have the form in there as well. Nice catch!

Updated!